app/models/authentication_htpasswd.rb

```diff
@@ -0,0 +1,12 @@
+class AuthenticationHtpasswd < Authentication
+
+  def ansible_config_format
+    super("filename" => "/etc/origin/master/htpasswd")
```

Comment:
Example of one where you're using a Hash literal, so the others can be Hash literals as well where appropriate.